carica_cfn_tools/stack_config.py

```diff
@@ -526,6 +526,7 @@ def _run_jinja_on_main_template(self, template_path):
             'deploy_bucket': self.bucket,
             # Prefix in the S3 bucket under which the template and its resources are deployed
             'deploy_prefix': f'{self.stack_name}/',
+            'utcnow': datetime.datetime.utcnow()
```

Comment:
I'd suggest passing a timezone to utcnow() so it's TZ-aware.

You'll also need to bump the version so we can publish to pypi and require the new version in arpio-api.